src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,28 @@
     list-style-type: none;
     padding: 0;
 }
+
+.launchpad-plugin .plugins .button.truncate-toggle, .scenarios .button.truncate-toggle {
```

Comment:
Minor, but consider adding a line break after the comma so it's easier to tell this style has 2 associated rules.